docs/learn/dapp-developer/siwe.md

```diff
@@ -14,14 +14,19 @@ Therefore, if the message you want to sign complies with this standard, the LUKS
 <div style={{textAlign: 'center'}}>
 
 <img
-    src="/img/learn/siwe.png"
```

Comment:
This iamge `/img/learn/siwe.png` seems to not be used anymore so let's delete it from the repo. I'll do it.